apps/etterlatte-behandling/src/main/kotlin/behandling/tilbakekreving/TilbakekrevingRoutes.kt

```diff
@@ -96,9 +96,31 @@ internal fun Route.tilbakekrevingRoutes(service: TilbakekrevingService) {
                 }
             }
         }
+
+        post("/oppgave-status") {
```

Comment:
Med fare for å vera pirkete, så synes eg både denne og den under godt kan vera patch eller eventuelt put.

Reply:
Endrer disse til put 👍